### ISSUE TITLE
All data labels for the training set and test set in NLI_M are None

### ISSUE THREAD

**Issue by cyzLoveDream** [HSLCY/ABSA-BERT-pair#11]
When I run the script generate_sentihood_NMI_M. py, the data's label is None. Is this the correct data format?
